assets/theme-supplement.css

```diff
@@ -3,6 +3,9 @@
   --sc-border-rounded: 7px;
   --sc-grid-gap: var(--sc-spacing-large);
   --sc-max-width: 1260px;
+  --sc-color-golden: #ffab41;
+  --sc-border-lightest: 1px solid rgba(128, 128, 128, 0.16);
+  --sc-spacing-base: 20px;
 }
```

Comment:
You don't need to re-declare it, it already exist in the core application.

Reply:
@gersongcg When I remove the declared root variables, the colors don't appear and the border.
![image](https://github.com/GetStoreConnect/clean-theme/assets/106049038/e703caf9-0485-4307-97f6-6a8c306e7e65)